src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings

```diff
@@ -151,3 +151,21 @@
 "TraceLocation_Information_Legal_Text_4" = "Please display the QR code in such a way that all your guests can scan it while observing the social distancing and hygiene rules (at least 1.5 metres).";
 
 "TraceLocation_Information_Legal_Text_5" = "We recommend creating new QR codes for places at regular intervals, so that warnings are as reliable as possible over time and in order to prevent the QR codes from being misused.";
+
+/* ExposureSubmission - AntigenTest Profile */
+
+"ExposureSubmission_Antigen_Profile_Legal_Headline" = "Data protection and data security";
+
+"ExposureSubmission_Antigen_Profile_Legal_Text01" = "Creating a rapid test profile in the app, and using it, are voluntary.";
```

Comment:
I'd say "is voluntary" instead of "are voluntary"

@janetback What do you think?

Reply:
Hi @Ein-Tim I see what you mean but this can be argued both ways, and since these texts are provided by the lawyers (both EN and DE) I suggest we leave the current wording here.